modules/packages/URL.chpl

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2004-2019 Cray Inc.
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+
+Download data from a URL or upload data to a URL.
+
+For example, the following program downloads a web-page from http://example.com and outputs each line to stdout:
+
+.. code-block:: chapel
+
+  use URL;
+  var urlreader = openUrlReader("http://example.com");
+  var str:string;
+  // Output each line read from the URL to stdout
+  while(urlreader.readline(str)) {
+    write(str);
+  }
+
```

Comment:
Is it possible to show a writer example too, or do you think that's a far rarer use-case and not needed?

Reply:
I wanted to keep the examples to ones we are able to test. The writer example is much rarer.